curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-pokemon-search-app-project/build-a-pokemon-search-app.md

```diff
@@ -14,22 +14,22 @@ In this project, you'll build an app that will search for Pokémon by name or ID
 
 **User Stories:**
 
-1. You should have an `input` element with an `id` of `search-input`
-1. You should have a `button` element with an `id` of `search-button`
-1. You should have an element with an `id` of `pokemon-name`
-1. You should have an element with an `id` of `pokemon-id`
-1. You should have an element with an `id` of `weight`
-1. You should have an element with an `id` of `height`
-1. You should have an element with an `id` of `types`
-1. You should have an element with an `id` of `hp`
-1. You should have an element with an `id` of `attack`
-1. You should have an element with an `id` of `defense`
-1. You should have an element with an `id` of `special-attack`
-1. You should have an element with an `id` of `special-defense`
-1. You should have an element with an `id` of `speed`
-1. When the `#search-input` element contains the value `Red` and the `#search-button` element is clicked, an alert should appear with the text `Pokémon not found`
+1. You should have an `input` element with an `id` of `"search-input"`
+1. You should have a `button` element with an `id` of `"search-button`
+1. You should have an element with an `id` of `"pokemon-name"`
+1. You should have an element with an `id` of `"pokemon-id"`
+1. You should have an element with an `id` of `"weight"`
+1. You should have an element with an `id` of `"height"`
+1. You should have an element with an `id` of `"types"`
+1. You should have an element with an `id` of `"hp"`
+1. You should have an element with an `id` of `"attack"`
+1. You should have an element with an `id` of `"defense"`
+1. You should have an element with an `id` of `"special-attack"`
+1. You should have an element with an `id` of `"special-defense"`
+1. You should have an element with an `id` of `"speed"`
+1. When the `#search-input` element contains the value `Red` and the `#search-button` element is clicked, an alert should appear with the text `"Pokémon not found"`
 1. When the `#search-input` element contains the value `Pikachu` and the `#search-button` element is clicked, the values in the `#pokemon-name`, `#pokemon-id`, `#weight`, `#height`, `#hp`, `#attack`, `#defense`, `#special-attack`, `#special-defense`, and `#speed` elements should be `PIKACHU`, `#25` or `25`, `Weight: 60` or `60`, `Height: 4` or `4`, `35`, `55`, `40`, `50`, `50`, and `90`, respectively
-1. When the `#search-input` element contains the value `Pikachu` and the `#search-button` element is clicked, you should add an `img` element with the `id` of `sprite` and the `src` set to the Pokémon's `front_default` sprite to the page
+1. When the `#search-input` element contains the value `Pikachu` and the `#search-button` element is clicked, you should add an `img` element with the `id` of `"sprite""` and the `src` set to the Pokémon's `front_default` sprite to the page
```

Comment:
```suggestion
1. When the `#search-input` element contains the value `Pikachu` and the `#search-button` element is clicked, you should add an `img` element with the `id` of `"sprite"` and the `src` set to the Pokémon's `front_default` sprite to the page
```

one quote only, not two

Reply:
Ooops, I will see to that as soon as 
Well spotted thanks